.circleci/config.yml

```diff
@@ -595,6 +595,7 @@ jobs:
           at: .
       - go-get-deps
       - setup_remote_docker
```

Comment:
I think you need a `:` after `setup_remote_docker`

Reply:
Ah yes, hence the immediate build error :) Thanks!